metadata-proxy/nginx.conf

```diff
@@ -0,0 +1,47 @@
+user www-data;
+worker_processes 4;
+pid /run/nginx.pid;
+
+events {
+  worker_connections 20;
+}
+
+http {
+  server {
+    listen 127.0.0.1:2020;
```

Comment:
Nothing was using it, and dynamic allocation would be considerably more difficult. Do you have another proposal?